shell/platform/embedder/embedder.h

```diff
@@ -251,6 +253,9 @@ typedef struct {
   const uint8_t* isolate_snapshot_instructions;
   // The size of the isolate snapshot instructions buffer.
   size_t isolate_snapshot_instructions_size;
+  // The callback invoked by the engine in root isolate scope. Called
+  // immediately after the root isolate has been created and marked runnable.
+  FlutterRootIsolateCreateCallback root_isolate_create_callback;
```

Comment:
Please use the VoidCallback as it already provides embedders the ability to provide a user data baton.

Reply:
Done.